google/cloud/security/notifier/pipelines/email_pipeline.py

```diff
@@ -0,0 +1,132 @@
+
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Email pipeline to perform notifications"""
+
+from datetime import datetime
+
+# TODO: Investigate improving so we can avoid the pylint disable.
+# pylint: disable=line-too-long,no-name-in-module
+from google.cloud.security.common.util import log_util
+from google.cloud.security.common.util import parser
+from google.cloud.security.common.util.email_util import EmailUtil
+from google.cloud.security.notifier.pipelines import base_notification_pipeline
+# pylint: enable=line-too-long,no-name-in-module
+
+LOGGER = log_util.get_logger(__name__)
+
+TEMP_DIR = '/tmp'
+VIOLATIONS_JSON_FMT = 'violations.{}.{}.{}.json'
+OUTPUT_TIMESTAMP_FMT = '%Y%m%dT%H%M%SZ'
+
+
+class EmailPipeline(base_notification_pipeline.BaseNotificationPipeline):
```

Comment:
QQ: Sorry I didn't think of this before.  This email pipeline is very specific to violations.  Do you think that it should be renamed to "EmailViolationsPipeline", and the module name becomes "email_violations_pipeline.py".  So that we can have room with other kinds of EmailFoobarPipeline?